cluster-provision/gocli/opts/rookceph/ceph_test.go

```diff
@@ -34,4 +36,18 @@ var _ = Describe("CephOpt", func() {
 		err := opt.Exec()
 		Expect(err).NotTo(HaveOccurred())
 	})
+
+})
+
+var _ = Describe("BackOff", func() {
+
+	It("backOff", func() {
+		backoffStrategy := backoff.NewExponentialBackOff()
+		backoffStrategy.InitialInterval = 30 * time.Second
```

Comment:
The initial interval here doesn't match the one in ceph.go. Was that intentional?

Reply:
Actually that was just a unit test to play around with the behavior of the factory func.

Reply:
I can remove the file, as the test doesn't have any practical value. WDYT?

Reply:
It has some didactic value, so I’d keep it, though I don’t have a strong opinion either way.